### PR TITLE
fix: deployment of OpenStack Neutron and Nova

### PR DESCRIPTION
**Pull request by cardoe**
Attempt to fix the templating to be able to successfully deploy Neutron and Nova. Added Placement since it seemed to be required by Nova.

### CODE REVIEW

File: components/openstack-secrets.tpl.yaml [hunk 1]
```diff
@@ -76,4 +142,13 @@ secrets:
     identity:
       api:
         public: keystone-tls-public
+    network:
+      api:
```
Comment (nicholaskuechler) [inline components/openstack-secrets.tpl.yaml:146]: should be `server`

**Review by nicholaskuechler — CHANGES_REQUESTED** [review]

**Comment by cardoe** [comment]
This gets nova and neutron in a running state with some basic defaults targeting a testing environment.

File: components/neutron/aio-values.yaml [hunk 2]
```diff
@@ -12,6 +12,39 @@ endpoints:
...
+    ml2_conf:
+      ml2:
+        # set the default ml2 backend as the logger for testing, actual deployment should override
+        mechanism_drivers: logger
+  neutron:
+    DEFAULT:
+      # we aren't using neutron routers
+      service_plugins: ""
+      # we don't want HA L3 routers
+      l3_ha: False
```
Comment (nicholaskuechler) [inline components/neutron/aio-values.yaml:36]: yaml lint doesn't like this... should this just be `false`? Or maybe `"False"`?

Comment (cardoe) [reply components/neutron/aio-values.yaml]: Yeah you're right. It's parsed in Python it turns out so it needed to be `"False"` like you suggested.

**Review by nicholaskuechler — APPROVED** [review]

**Pull request merged**